initrd/bin/reboot

```diff
@@ -3,14 +3,23 @@
 
 TRACE "Under /bin/reboot"
 
+if [ "$CONFIG_DEBUG_OUTPUT" = "y" ]; then 
+	#Generalize user prompt to continue reboot or go to recovery shell 
+	read -r -n 1 -s -p "Press any key to continue reboot or 'r' to go to recovery shell: " REPLY 
+	echo 
+	if [ "$REPLY" = "r" ] || [ "$REPLY" = "R" ]; then 
+		recovery "Reboot call bypassed to go into recovery shell to debug"
+	fi 
+fi
+
```

Comment:
@JonathonHall-Purism this is the code now called upon reboot, and permits to enter recovery shell instead to verify things when in debug mode. We would expect "recovery" to call gpg_auth in current state, which set user_config permit to test authentication without having to rewrite board config for that signle test, but it doesn't work, which is why I'm asking for advice here.

Reply:
Thanks, I missed this.  Commented elsewhere